admin/src/router/guards.js

```diff
@@ -0,0 +1,12 @@
+const addNavigationGuards = (router, store) => {
+  router.beforeEach((to, from, next) => {
+    // handle authentication
```

Comment:
i suggest:
```suggestion
    // handle authentication
    if( to === '/authenticate' && req.post.token){
       store.commit('token',req.post.token)
    }
```
On the `/authenticate` I suggest to check for a valid token in the store and redirect again according to this finding.